guides/using-iron-icons.md

```diff
@@ -0,0 +1,243 @@
+size. It can be bitmap or SVG. `iron-icon` expects the source image to be square.
+The default icon size is 24 pixels by 24 pixels. Use CSS to set the icon size.
+
+    ...
+    <iron-icon src="/images/polymer.svg" style="width: 24px; height: 24px;"></iron-icon>
+    <iron-icon src="/images/polymer.svg" style="width: 32px; height: 32px;"></iron-icon>
+    <iron-icon src="/images/polymer.svg" style="width: 48px; height: 48px;"></iron-icon>
+    ...
+
+Produces: 
+
+<iron-icon src="/images/polymer.svg" style="width: 24px; height: 24px;"></iron-icon>
+<iron-icon src="/images/polymer.svg" style="width: 32px; height: 32px;"></iron-icon>
+<iron-icon src="/images/polymer.svg" style="width: 48px; height: 48px;"></iron-icon>
+
+### Globally styling width and height
+
+To create style rules that affect the size of all icons, create a theme file
+and use the `--iron-icon-width` and `iron-icon-height` mixins.
+
+```html
+<!-- custom-icon.html -->
+<style is="custom-icon">
+  iron-icon {
+    --iron-icon-width: 100px;
+    --iron-icon-height: 100px;
+  }
+</style>
+```
+
+Import the theme file into your page and style the icons with your custom class:
+
+```html
+   ...
+   <link rel="import" href="custom-icon.html">
+   ...
+   <iron-icon class="custom-icon" icon="accessibility"></iron-icon>
+```
+
+## Using the Polymer team's icon collection, `iron-icons`
+
+The Polymer team has created a large collection of free, SVG icons that 
+you can use in your own project. These icons are distributed as an element
+called `iron-icons`. 
+
+[View all of the icons here.](https://elements.polymer-project.org/elements/iron-icons?view=demo:demo/index.html)
+
+To use one of the Polymer team's icons, import the `iron-icons` element 
+into your project and reference the icon via the `icon` attribute:
+
+    ...
+    <!-- iron-icons loads the default icon set and the iron-icon element -->
+    <link rel="import" href="/bower_components/iron-icons/iron-icons.html">
+    ...
+    <iron-icon icon="refresh"></iron-icon>
+    ...
+
+
+Produces:
+
+<iron-icon icon="refresh"></iron-icon>
+
+Note that you reference the image source via the `icon` attribute, not
+the `src` attribute, as is the case when you using an image that is not part
+of an icon set.
+
+The default icon set is called `icons`. If you reference an icon 
+using only the icon name, `iron-icon` will search for the name within 
+the `icons` icon set. For example, the two declarations below reference the 
+same icon.
+
+    <iron-icon icon="refresh"></iron-icon>
+    <iron-icon icon="icons:refresh"></iron-icon>
+
+`iron-icons` contains many more icons, grouped into the following 
+thematic sets: 
+
+* Audio Visual
+* Communication
+* Device
+* Editor
+* Hardware
+* Images
+* Maps
+* Notification
+* Social
+* Update
+
+To use an icon from one of these groups, import the icon set and then reference
+the icon using the following syntax:
+
+    <icon set>:<icon name>
```

Comment:
The rest of the doc set uses italics for variables/placeholders. MD doesn't handle this gracefully, so I do it in HTML:

    <code><var>icon set</var>:<var>icon name</var></code>

If it makes the text more clear, you can bold the literal characters (in this case, just the ":"), but I haven't done that consistently.